sdk/src/contexts/solana/context.ts

```diff
@@ -953,8 +950,10 @@ export class SolanaContext<
           parsed.tokenChain,
           parsed.tokenAddress,
         );
-    const transaction = new Transaction();
-    transaction.add(...(await this.determineComputeBudget([mint])));
+
+    const transaction = new Transaction(
+      await this.connection?.getLatestBlockhash('finalized'),
```

Comment:
why finalized? confirmed might give you a couple more seconds but generally we should probably not set these on a per call basis

Reply:
I just maintained what was there. check the diff.

<img width="816" alt="image" src="https://github.com/wormhole-foundation/wormhole-connect/assets/897596/0ac56cba-bf81-4526-b6cb-9105134cbee4">

Reply:
sure, but since we're here trying to make it work gooder, we should consider this up for debate

I'm positive @nonergodic has experience with varying consistency levels

Reply:
Yeah, the general rule (though ofc there are exceptions) is that you don't want to override the commitment level of the connection (unless you really know what you are doing). It's really quite the footgun in the Solana SDK due to how juicy and tempting it is to just muck with it and suddenly all hell breaks loose, especially when someone changes the default commitment level on the connection.

This case seems to be a good example of that. If you call `getLatestBlockhash` with finalized commitment, then I'd expect that blockhash to be comparatively old (32 slots) which can then lead to it timing out faster. I'm not sure if it is really a problem here in particular or whether this is the right approach (I'm mostly working on the on-chain side of things and so my knowledge of the solana web3 sdk and interacting with the chain via a node/validator are somewhat limited) but I have seen various related instances where somebody was like "I know, I'll just fetch the latest blockhash using `processed` commitment level because that will give me the newest one!" and then they tried to submit that transaction with a more robust commitment level and the Solana said "I don't know this blockhash." (when it should have said "I don't know this blockhash at the more robust commitment level you have selected.").

Reply:
an old example of these sorts of issues that came to my mind:
https://github.com/XLabs/portal-bridge-ui/pull/154#issuecomment-1558071589